sql/mozfun/norm/get_earliest_value/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Get Earliest Value
+description: |
+  This UDF returns the earliest not-null value from a set of 3 values.
+
+  The order is not important as the function will return the first value
```

Comment:
This isn't the case for us, right? We want to take the first value for a given date. For example, we want to first get the value from the `first_session` ping on a day, and if unavailable, get it from the `metrics` ping on that day.

Reply:
@bani here is the scenario I explained, what do you think?

Reply:
Usually I expect first_session to come before metrics, but if it happens that metrics comes first, I'm fine with that value, we can use whatever ping is available earliest that contains non-NULL data.

Reply:
After checking with Vanessa, we confirmed about collecting the first reported (not null) value, from any ping,